### PR TITLE
Make formatters file more readable

### PR DESCRIPTION
**Pull request by xavier630**
This is a minor change to help make the formatters file more readable.

### CODE REVIEW

File: code/formatters.py [hunk 1]
```diff
@@ -12,6 +12,11 @@
     ","
 )
 
+DEFAULT_SEPARATOR = ' '
```
Comment (rntz) [inline code/formatters.py:15]: I'm not convinced these globals (`DEFAULT_SEPARATOR`, `SMASH_SEPARATOR`) improve readability. They're only used in one place, so this just separates their definition from where they're used.

File: code/formatters.py [hunk 2]
```diff
@@ -12,6 +12,11 @@
...
+SMASH_SEPARATOR = ''
+SEP = True
```
Comment (rntz) [inline code/formatters.py:17]: While we're cleaning things up, SEP and NOSEP could use a comment explaining what they mean/do.

**Review by rntz — APPROVED** [review]

**Comment by rntz** [comment]
If anyone merges this, please squash-merge to keep the commit history uncluttered.

File: code/formatters.py (hunk 2 above)
Comment (xavier630) [reply code/formatters.py:17]: https://github.com/knausj85/knausj_talon/pull/746

File: code/formatters.py (hunk 1 above)
Comment (xavier630) [reply code/formatters.py:15]: Have left this as is, but if you feel strongly, we can easily adjust 👍

**Pull request merged**